app/controllers/forgot_passwords_controller.rb

```diff
@@ -0,0 +1,26 @@
+class ForgotPasswordsController < ApplicationController
+
+  def create
+    user = User.where(email: params[:email]).first
+    if user
+      @token = user.generate_token
+      user.update_column(:token, @token)
```

Comment:
This could be part of the generate_token method and update the column inside the model directly.